web/html/src/utils/functions.ts

```diff
@@ -116,16 +116,34 @@ function capitalize(str: string): string {
 }
 
 function generatePassword(): string {
-  const length = Math.floor(Math.random() * 10) + 15;
-  // See https://stackoverflow.com/a/68617567/1470607
-  const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789:-_";
-  const array = new Uint32Array(charset.length);
-  window.crypto.getRandomValues(array);
+  const length = 24; // fixed length for better security
```

Comment:
I think this is not true? Using a fixed length reduces security, since it makes the space you need to brute force smaller. To exemplify, if we use only passwords of length 25, then there are `X^25` combinations where `X` is the number of symbols in our charset, but if we allow for example passwords of length 15-25, then there are `X^15 + X^16 + X^17 ... X^25` combinations.